### PR TITLE
feat: shift + click to expand all child components

### LINKED ISSUE #1739: alt-click to recursively expand component tree
### Version
6.0.9

### Browser and OS info
chrome LTS, windows 10

### Steps to reproduce
open vue dev tools, go to Components tab, alt-click on arrow next to component, see that tree does NOT recursively open

### What is expected?
tree to recursively open (was standard on devtools before Vue3 became default)

### What is actually happening?
just opens 1 layer

---
please bring this feature back

### PR DESCRIPTION
**Pull request by konpeki622**
closes #1739

force update: merged `forceToggleAll` to `toggle`

### CODE REVIEW

File: packages/app-frontend/src/features/components/composable/components.ts [hunk 1]
```diff
@@ -136,11 +136,18 @@ export function useComponent (instance: Ref<ComponentTreeNode>) {
...
-    setComponentOpen(instance.value.id, !isExpanded.value)
-    if (isComponentOpen(instance.value.id)) {
-      requestComponentTree(instance.value.id)
+  function toggleExpand (recursively = false, value?, child?) {
+    const treeNode = child || instance.value
+    if (!treeNode.hasChildren) return
+    const isOpen = value === undefined ? !isExpanded.value : value
+    setComponentOpen(treeNode.id, isOpen)
+    if (isComponentOpen(treeNode.id)) {
+      requestComponentTree(treeNode.id)
```
Comment (Akryum) [inline packages/app-frontend/src/features/components/composable/components.ts:145]: We would also need to expand the nodes that are loaded after that (recursively).

Comment (konpeki622) [reply packages/app-frontend/src/features/components/composable/components.ts]: I added property `autoOpen` to `TreeNode` to resolve it.

**Comment by Akryum** [comment]
Would you mind allowing us to push changes to your branch?

**Comment by konpeki622** [comment]
@Akryum That's ok.

**Comment by Akryum** [comment]
I mean I don't think we technically have the necessary rights to be able to push to this branch.

**Comment by konpeki622** [comment]
I have checked allowing edits. Still can not push to it?👀

**Comment by Akryum** [comment]
Let me try again

**Comment by Akryum** [comment]
Great it works :)
